extjsdk/src/main/java/io/vantiq/extjsdk/ExtensionWebSocketListener.java

```diff
@@ -386,7 +416,7 @@ else if (msg.get("op").equals(ExtensionServiceMessage.OP_CONFIGURE_EXTENSION) &&
                 }
             }
             else {
-                log.warn("ExtensionServiceMessage with unknown/unexpected op '" + msg.get("op") + "'");
+                log.warn("ExtensionServiceMessage received when not connected");
```

Comment:
(general) -- local code format does
if (...) {
    ....
} else if (...) {
   ...
} else {
    ....
}

that is, else/else-if on same line as previous closing brace.  Not required for this review, but as going through...